vmdb/app/views/layouts/listnav/_ems_container.html.haml

```diff
@@ -85,3 +85,13 @@
               = link_to(_("Replicators (%s)") % num_replicators,
                   {:action  => 'show', :id => @record, :display => 'container_replicators'},
                   :title => _("Show Replicators"))
+        - if role_allows(:feature => "container_image_registry_show_list")
```

Comment:
@zeari this is not working for me. If you go in the provider page it crashes on missing `container_image_registries`